src/components/editor/faultTree/menu/faultTree/FaultTreeContextMenu.tsx

```diff
@@ -35,7 +36,7 @@ const FaultTreeContextMenu = ({ anchorPosition, onClose, onEditClick, onDelete }
           : undefined
       }
     >
-      <MenuItem key="fault-tree-menu-rename" onClick={handleEditClick}>
+      <MenuItem disabled={!canRename} key="fault-tree-menu-rename" onClick={handleEditClick}>
```

Comment:
Is it possible to put a tooltip that "FHA-based events cannot be renamed?"

Reply:
@kostobog

Reply:
@blcham 
Yes, I think it is possible.

Reply:
I created a new ticket for that -- https://github.com/kbss-cvut/fta-fmea-ui/issues/551